src/network/clientpackethandler.cpp

```diff
@@ -1470,13 +1470,18 @@ void Client::handleCommand_CloudParams(NetworkPacket* pkt)
 	f32 density;
 	video::SColor color_bright;
 	video::SColor color_ambient;
+	video::SColor color_shadow;
```

Comment:
default needs to be handled here somehow or it will default to something else when the server doesn't send a value